pandas/core/sparse/series.py

```diff
@@ -175,7 +175,7 @@ def values(self):
 
     def __array__(self, result=None):
         """ the array interface, return my values """
-        return self.block.values
+        return self.block.values.values
```

Comment:
This will return the dense version of SparseSeries when calling `np.array(sp)`

Reply:
we need some more asv's for sparse I think